tests/acceptance/features/apiCollaboration/wopi.feature

```diff
@@ -432,3 +432,25 @@ Feature: collaboration (wopi)
       | app-endpoint                                              | url-query       |
       | /app/open-with-web?file_id=<<FILEID>>&app_name=FakeOffice | app=FakeOffice& |
       | /app/open-with-web?file_id=<<FILEID>>                     |                 |
+
+
+  Scenario: create text file using app endpoint
```

Comment:
How add adding tests for other drive space/share if possible.

Reply:
I think for text file that would not be necessary.

Reply:
in API tests, we cover every edge case. So, IMO covering every type of drive goes with our principle until there is limitation.